provider-utils/awscloudformation/cloudformation-templates/vod-helpers/LambdaFunctions/OutputLambda/index.js.ejs

```diff
@@ -1,4 +1,3 @@
-es (31 sloc) 819 Bytes
```

Comment:
assuming this is something that got into the project via a build step? a little (any) info in #85 would help me understand what to review in this ;)